plugin/src/main/kotlin/com/ritense/plugin/service/PluginService.kt

```diff
@@ -119,12 +119,13 @@ class PluginService(
         ) {
             deletePluginConfiguration(PluginConfigurationId.existingId(deploymentDto.id))
         }
+        resolveProperties(deploymentDto.properties)
```

Comment:
resolvesProperties returns a new ObjectNode. It does not change the original deploymentDto.properties.

This change actually breaks the property resolving.